website/content/en/docs/config/environment-variables.md

```diff
@@ -77,3 +77,15 @@ This page documents the environment variables used in Lima.
   ```
 - **Note**: It is expected that this variable will be set to `false` by default in future
   when QEMU supports `pflash` UEFI for accelerated guests on Windows.
+
+### `_LIMA_WINDOWS_EXTRA_PATH`
```

Comment:
Why prepend `_`  ?

Reply:
It was requested by @jandubois in https://github.com/lima-vm/lima/issues/3332#issuecomment-2711587045 to highlight the experimental nature of this setting.

> I would probably prefix the variable with _LIMA instead of LIMA to make it obvious that this is an experimental thing.

Reply:
`LIMA_QEMU_UEFI_IN_BIOS` is also kinda experimental but it doesn't have `_`

Reply:
Either would be fine, but they have to be consistent

Reply:
Renamed the other one to make it consistent.